index-handlers/src/test/java/no/sikt/nva/nvi/index/IndexNviCandidateHandlerTest.java

```diff
@@ -19,84 +22,129 @@
 import com.amazonaws.services.lambda.runtime.events.models.dynamodb.AttributeValue;
 import com.amazonaws.services.lambda.runtime.events.models.dynamodb.OperationType;
 import com.amazonaws.services.lambda.runtime.events.models.dynamodb.StreamRecord;
-import java.io.IOException;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import java.net.URI;
 import java.nio.file.Path;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import no.sikt.nva.nvi.common.StorageReader;
+import no.sikt.nva.nvi.common.model.CandidateWithIdentifier;
+import no.sikt.nva.nvi.common.model.business.ApprovalStatus;
+import no.sikt.nva.nvi.common.service.NviService;
 import no.sikt.nva.nvi.index.aws.OpenSearchClient;
 import no.sikt.nva.nvi.index.aws.SearchClient;
 import no.sikt.nva.nvi.index.model.NviCandidateIndexDocument;
+import no.sikt.nva.nvi.test.LocalDynamoTest;
+import no.unit.nva.commons.json.JsonUtils;
+import nva.commons.core.StringUtils;
 import nva.commons.core.ioutils.IoUtils;
 import nva.commons.logutils.LogUtils;
 import nva.commons.logutils.TestAppender;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-class IndexNviCandidateHandlerTest {
+class IndexNviCandidateHandlerTest extends LocalDynamoTest {
```

Comment:
Add tests that actually test the data in the IndexDocument? See one of my earlier commits [index-handlers/src/test/java/no/sikt/nva/nvi/index/IndexNviCandidateHandlerTest.java ](https://github.com/BIBSYSDEV/nva-nvi/pull/3/files#diff-ed44306bd20c2c3e8bb267df432c6a423e6ecdb44ada7ee6e7bf0e7f46144ff6)